conjure-core/src/main/java/com/palantir/conjure/defs/ConjureParserUtils.java

```diff
@@ -191,6 +193,25 @@ public static Type parsePrimitiveType(com.palantir.conjure.parser.types.primitiv
         return Type.primitive(PrimitiveType.valueOf(primitiveType.name()));
     }
 
+    static Type parseExternalType(ExternalTypeDefinition externalType, String conjurePackage, String typeName) {
+        if (isBearerToken(externalType)) {
```

Comment:
I think this will fail if any external types declare base-type bearer-token
```suggestion
        if (isBearerToken(externalType) && externalType.safety().isPresent()) {
```

Reply:
oops, thanks for the catch